native/package.json

```diff
@@ -120,10 +120,12 @@
     "jest-transform-stub": "^2.0.0",
     "jetifier": "^1.6.5",
     "metro-react-native-babel-preset": "^0.59.0",
+    "react-native-flipper": "^0.62.0",
```

Comment:
What does this dependency do?

Reply:
It is required by the redux-flipper plugin which allows you to see actions and state in flipper